cups/dest.c

```diff
@@ -3688,6 +3694,8 @@ cups_enum_dests(
   else
     remaining = msec;
 
+  data.num_dests = cups_get_dests(filename ,NULL, NULL, 1, user_default != NULL, data.num_dests, &data.dests);
```

Comment:
@pranjanpr there mustn't be a space between an argument and a comma, the space must be after the comma.